src/ImageService.Data.Provider.MsSql.Ef/Migrations/20210820233300_InitialTable.cs

```diff
@@ -35,9 +34,9 @@ protected override void Up(MigrationBuilder builder)
                 {
                     Id = table.Column<Guid>(nullable: false),
                     ParentId = table.Column<Guid>(nullable: true),
-                    Name = table.Column<string>(nullable: true),
+                    Name = table.Column<string>(nullable: false),
                     Content = table.Column<Guid>(nullable: true),
```

Comment:
content is string, nullable is false
name - nullable is true

Reply:
done